.github/workflows/lgtm-merger.yaml

```diff
@@ -0,0 +1,17 @@
+name: "Merge on lgtm label"
+on:
+  schedule:
+    - cron: "*/20 * * * *"
+
+jobs:
+  execute:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
These action steps are for the original prow action source I believe. Users of the action need to use the onPr version of the workflow that has been added below.